lib/src/main/java/com/orange/ods/compose/component/navigationdrawer/OdsModalDrawer.kt

```diff
@@ -0,0 +1,253 @@
+                contentScale = ContentScale.Crop,
+                modifier = headerContent.modifier
+                    .fillMaxWidth()
+            )
+            Surface(
+                color = Color.Black.copy(alpha = 0.8f),
+                modifier = Modifier
+                    .align(Alignment.BottomStart)
+                    .fillMaxWidth()
+            ) {
+                OdsHeaderText(headerContent = headerContent, color = Color.White)
+            }
+        }
+        headerContent.avatar?.let {
+            OdsImageCircleShape(
+                painter = it,
+                Modifier.padding(start = dimensionResource(id = R.dimen.spacing_m), top = 40.dp)
+            )
+            Column(Modifier.align(Alignment.BottomStart)) {
+                OdsHeaderText(headerContent = headerContent, color = Color.Black)
+            }
+
+        }
+        if (headerContent.backgroundImage == null && headerContent.avatar == null) OdsHeaderText(headerContent = headerContent, color = Color.Black)
+    }
+}
+
+@Composable
+fun OdsHeaderText(headerContent: OdsModalDrawerHeaderParametersProvider, color: Color) {
+    Column(
+        Modifier
+            .padding(start = dimensionResource(id = R.dimen.spacing_m), top = 12.dp, bottom = 12.dp)
+    ) {
+        Text(text = headerContent.title, color = color, style = OdsTheme.typography.h6)
+        headerContent.subtitle?.let {
+            Text(text = it, color = color, style = OdsTheme.typography.body2)
+        }
+    }
+}
+
+@UiModePreviews.Default
+@Composable
+private fun PreviewOdsModalDrawer(@PreviewParameter(OdsModalDrawerPreviewParameterProvider::class) parameter: OdsModalDrawerPreviewParameter) =
+    Preview {
+        OdsModalDrawer(
+            drawerState = rememberDrawerState(DrawerValue.Open),
+            content = {},
+            headerParametersProvider = OdsModalDrawerHeaderParametersProvider(
+                title = parameter.title,
+                subtitle = parameter.subtitle,
+                avatar = parameter.avatar?.let { painterResource(id = it) },
+                backgroundImage = parameter.backgroundImage?.let { painterResource(id = it) },
+            ),
+            listContent = parameter.list
+        )
+    }
+
+private data class OdsModalDrawerPreviewParameter(
+    val backgroundImage: Int?,
+    val title: String,
+    val avatar: Int?,
+    val subtitle: String?,
+    val list: List<OdsModalDrawerItem>
+)
+
+private class OdsModalDrawerPreviewParameterProvider :
+    BasicPreviewParameterProvider<OdsModalDrawerPreviewParameter>(*previewParameterValues.toTypedArray())
+
+private val previewParameterValues: List<OdsModalDrawerPreviewParameter>
+    get() {
+        val backgroundImage = R.drawable.placeholder_small
+        val title = "Headline 6"
+        val avatar = R.drawable.placeholder
+        val icon = R.drawable.ic_check
+        val subtitle = "Body 2"
+        val noPicture = null
```

Comment:
Instead of defining a variable to `null`, use `null` directly. It makes the code readable: `subtitle = null` is more readable than `subtitle = noPicture` ;)